format_standardization.txt

```diff
@@ -1,65 +0,0 @@
-***************************************************
```

Comment:
Can this file just be deleted?

Reply:
the file has been deleted. You can check the branch `martin/refactor_struct`. I moved it to the wiki page